blast/templates/blast/main.html

```diff
@@ -35,7 +35,7 @@
 <div class="tripal_organism-info-box tripal-info-box">
 <!--<a href="docs/blast_databases.html">-->
 <fieldset id="fieldset-datasets">
-    <legend>BLAST Databases <a href="https://i5k.nal.usda.gov/content/blast-tutorial" target="_blank"><span id="tutorial"><span class="glyphicon glyphicon-book"></span> Tutorial</span></a></legend>
+    <legend>BLAST Databases <a href="https://i5k.nal.usda.gov/webapp/blast/manual/" target="_blank"><span id="tutorial"><span class="glyphicon glyphicon-book"></span> Tutorial</span></a></legend>
```

Comment:
The thing needs to be considered here is that this website project is intended for general purpose. We don't want to hard-coded this. Instead, we want make this link go to the url `<host>/blast`.